hardhat.config.ts

```diff
@@ -37,17 +37,55 @@ task('accounts', 'Prints the list of accounts', async (taskArgs, hre) => {
     console.log(account.address)
   }
 })
+const networkFork = process.env.NETWORK_FORK
 
-const blockNumber = process.env.BLOCK_NUMBER
-if (!blockNumber) {
-  throw new Error(`You must provide a block number.`)
+if (!networkFork && (networkFork == 'Mainnet' || networkFork == 'Optimism')) {
```

Comment:
```suggestion
if (!networkFork || !(networkFork == 'Mainnet' || networkFork == 'Optimism')) {
```
Was that the intention?